content/en/getting-started/faq.md

```diff
@@ -188,3 +188,19 @@ $ dig api.localstack.cloud
 If the result has some other status than `status: NOERROR,` your machine cannot resolve this domain.
 
 Some corporate DNS servers might filter requests to certain domains. Contact your network administrator to safelist` localstack.cloud` domains.
+
+### What's LocalStack Pro's patching schedule for security patches and bug fixes?
+
+Emergency vulnerabilities are treated with the highest priority and fixed on the same business day. The cloud provider (AWS) manages and maintains most of the infrastructure. All infrastructure configuration is fully encoded as code scripts (Infrastructure-as-Code) and can quickly be recreated in an emergency.
```

Comment:
We take security seriously and respond to any emergency vulnerabilities as soon as possible. Our cloud provider (AWS) handles most of the infrastructure maintenance for us. We also use Infrastructure-as-Code scripts to ensure that our infrastructure configuration is consistent and recoverable in case of a disaster.